apps/meteor/app/ui/client/lib/KonchatNotification.ts

```diff
@@ -56,7 +58,7 @@ class KonchatNotification {
 		const requireInteraction = getUserPreference<boolean>(Meteor.userId(), 'desktopNotificationRequireInteraction');
 		const n = new Notification(notification.title, {
 			icon: notification.icon || getUserAvatarURL(notification.payload.sender?.username as string),
-			body: stripTags(message.msg),
+			body: notification.reacted ? parseReaction(notification.text, notification.payload.type as RoomType) : stripTags(message.msg),
```

Comment:
we are actually sending reaction text from the backend and parsing it in the client.